src/horde_map.cpp

```diff
@@ -89,15 +103,29 @@ std::vector<std::unordered_map<tripoint_abs_ms, horde_entity> *> horde_map::enti
     if( dormant_monster_map_iter != dormant_monster_map.end() ) {
         horde_chunk.push_back( &dormant_monster_map_iter->second );
     }
+    auto immobile_monster_map_iter = immobile_monster_map.find( p );
+    if( immobile_monster_map_iter != immobile_monster_map.end() ) {
+        horde_chunk.push_back( &immobile_monster_map_iter->second );
+    }
     return horde_chunk;
 }
 
+// Helper because this is too much to inline.
+// This essentially means "these should get sounds broadcast to them".
+static bool is_alert( const mtype &type )
+{
+    bool aggro = type.in_species( species_ZOMBIE ) || type.in_species( species_FERAL );
```

Comment:
This could just be a property of the `species_type`, instead of limiting by id.

Reply:
I'd rather not extend the species struct in a hurry for a targeted and ephemeral bugfix like this.
Like this filtering is going to be invalidated soon:tm: by other extensions to horde handling.